netlogo-gui/src/main/lab/gui/RunOptionsDialog.scala

```diff
@@ -16,9 +16,13 @@ class RunOptionsDialog(parent: java.awt.Dialog,
     private val prefs = Preferences.userNodeForPackage(RunOptionsDialog.this.getClass)
     def spreadsheet = prefs.getBoolean("spreadsheet", true)
     def table = prefs.getBoolean("table", false)
+    def updateView = prefs.getBoolean("updateView",true)
+    def updatePlotsAndMonitors = prefs.getBoolean("updatePlotsAndMonitors",true)
     def updateFrom(runOptions: RunOptions): Unit = {
       prefs.putBoolean("spreadsheet", runOptions.spreadsheet)
       prefs.putBoolean("table", runOptions.table)
+      prefs.putBoolean("updateView",runOptions.updateView)
+      prefs.putBoolean("updatePlotsAndMonitor",runOptions.updatePlotsAndMonitors)
```

Comment:
Typo, should be "updatePlotsAndMonitor**s**"

Reply:
thanks , pushing the fix